core/vote/vote_journal.go

```diff
@@ -0,0 +1,100 @@
+package vote
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/tidwall/wal"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/metrics"
+)
+
+const (
+	maxSizeOfRecentEntry = 512
```

Comment:
why this number? any calculation, any safe guarantee for using this number?

Reply:
This is just the max limitation to make sure we can always get votes from journal whenever we want, because it must be finalized within 512 blocks